packages/evolution-common/src/services/baseObjects/attributeTypes/PersonAttributes.ts

```diff
@@ -70,25 +76,98 @@ export type RidesharingUser = YesNoDontKnowNonApplicable;
 
 // A person can have multiple occupations, like fullTimeStudent + partTimeWorker:
 export const occupationValues = [
-    'fullTimeWorker',
-    'partTimeWorker',
-    'fullTimeStudent',
-    'partTimeStudent',
-    'workerAndStudent',
+    'fullTimeWorker', // should be assigned from WorkerType
+    'partTimeWorker', // should be assigned from WorkerType
+    'fullTimeStudent', // should be assigned from StudentType
+    'partTimeStudent', // should be assigned from StudentType
+    'workerAndStudent', // deprecated, not precise enough, see below:
+    'workerFullTimeAndStudentFullTime', // should be assigned from WorkerType && StudentType
+    'workerFullTimeAndStudentPartTime', // should be assigned from WorkerType && StudentType
+    'workerPartTimeAndStudentFullTime', // should be assigned from WorkerType && StudentType
+    'workerPartTimeAndStudentPartTime', // should be assigned from WorkerType && StudentType
     'retired',
+    'parentalOrSickLeave', // should be assigned from WorkerType, label must be clear
     'atHome',
     'unemployed',
     'other',
-    'nonApplicable',
-    'preferNotToAnswer'
+    'dontKnow', // not always available
+    'nonApplicable', // assign automatically, not shown in choices
+    'preferNotToAnswer' // not always available
 ];
 export type Occupation = (typeof occupationValues)[number];
 
+export const schoolTypeValues = [
+    'childcare',
+    'kindergarten',
+    'primarySchool',
+    'secondarySchool',
+    'kindergartenFor4YearsOld',
+    'collegeProfessional', // includes CEGEP, DEP, AEP or other professional diploma
+    'university',
+    'schoolAtHome',
+    'noSchool',
+    'other',
+    'dontKnow', // not always available
+    'nonApplicable', // assign automatically, not shown in choices
+    'preferNotToAnswer' // not always available
+];
+export type SchoolType = (typeof schoolTypeValues)[number];
+
+export const fullTimePartTimeTypeNoValue = [
+    'fullTime',
+    'partTime',
+    'no',
+    'dontKnow',
+    'nonApplicable'
+];
+export type StudentType = (typeof fullTimePartTimeTypeNoValue)[number];
+
+export const schoolPlaceTypeValues = [
+    'onLocation',
+    'hybrid',
+    'remote'
+];
+export type SchoolPlaceType = (typeof schoolPlaceTypeValues)[number];
+
+export type WorkerType = (typeof fullTimePartTimeTypeNoValue)[number] | 'parentalOrSickLeave';
```

Comment:
On peut être en maladie ou congé parental et être étudiant et non travailleur, non? Ici, ça semble assumer que parentalOrSickLeave implique travailleur

Reply:
If we allow an array, it is a big change in survey data... This needs discussions. Let's keep it as a single value for now.